core-bundle/src/Resources/contao/themes/flexible/src/basic.css

```diff
@@ -539,7 +541,7 @@ a.tl_submit {
 	margin:16px 15px 8px;
 }
 .cbx {
-	min-height:56px;
+	min-height:57px;
```

Comment:
This increased the overall height of these fields to 81px, although the text and select widgets are only 80px high. Tested on Mac OS with Safari, Chrome and Firefox.